go/libkb/mounter.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
```

Comment:
can you move these files into a different dir, maybe ../mounter?  libkb is getting super crowded...